api/routers/config.py

```diff
@@ -98,6 +98,7 @@ async def update_config_file(config: ConfigDTO, reboot_processor: Optional[bool]
 
 @config_router.get("/info", response_model=ConfigInfo)
 async def get_processor_info():
+    # Here we have to specify the camera right? Or are we asking for the default values?
```

Comment:
This part only cares about the device name, so it is a default value. It asks for the model name but just to recognize openvino.

Reply:
It is still coherent that the endpoint asks if the default model name is "openvino"?

Reply:
Yes. As the only  model name available for openvino is "openvino". The filepath my change but it is still openvino

Reply:
Okey, perfect!